signatures/gemss/BlueGemss/BlueGemss.yaml

```diff
@@ -0,0 +1,3 @@
+name: GeMSS
```

Comment:
Where is the documentation what flavours are for?

Reply:
Yeah, R&S have already mentioned this, there's indeed a lack of documentation here.

Basically, what we thought about when making up the separation was that there's things that are
- more different than just parameter sets
- but still belong together somehow

for example, because they have a differing API (for example a passively secure PKE (KTM) and an actively secure KEM built upon the former), or completely different and incompatible implementations (for example, one scheme using AES and one using Shake). The latter is not a harsh definition, this could also be implemented as parameter sets, but we thought it's clearer when we call them flavors).

Because the red/blue/vanilla GeMSS schemes only differ in the numbers, it's a quite clear case of parameter sets. What is different between them is just how strict they are in the qubits<->category relation. (Personally, I'd expect NIST to opt for one "color" (=calculation method) and throw out the rest anyways)

Reply:
If you can do it easily, I would prefer if you do it.
The other changes are done.